CMakeLists.txt

```diff
@@ -45,7 +45,7 @@ endif()
 option(ENABLE_CUBLAS_BACKEND "Enable the cuBLAS backend for the BLAS interface" OFF)
 option(ENABLE_ROCBLAS_BACKEND "Enable the rocBLAS backend for the BLAS interface" OFF)
 option(ENABLE_NETLIB_BACKEND "Enable the Netlib backend for the BLAS interface" OFF)
-option(ENABLE_PORTBLAS_BACKEND "Enable the portBLAS backend for the BLAS interface. Cannot be used with other BLAS backends." OFF)
+option(ENABLE_GENERIC_BLAS_BACKEND "Enable the generic BLAS backend for the BLAS interface. Cannot be used with other BLAS backends." OFF)
```

Comment:
Can you add some logic below to check if the user defines `ENABLE_PORTBLAS_BACKEND`? I suggest that we add a warning message explaining is `ENABLE_PORTBLAS_BACKEND` deprecated. We can automatically set `ENABLE_GENERIC_BLAS_BACKEND=ON` in this case to keep the internal CI working.